ovis_bringup/launch/ovis_arm.launch

```diff
@@ -13,14 +13,16 @@
     <param name="robot_type" value="$(arg kinova_robotType)" />
     <param name="use_jaco_v1_fingers" value="$(arg use_jaco_v1_fingers)" />
     <param name="status_interval_seconds" value="$(arg feedback_publish_rate)" />
-    <param name="number_of_degree_per_sec" value="5" />
-    <param name="loop_per_command" value="50" />
-    <param name="home_position_actuator1" value="18.703125" />
-    <param name="home_position_actuator2" value="174.750000" />
-    <param name="home_position_actuator3" value="94.031250" />
-    <param name="home_position_actuator4" value="331.909088" />
-    <param name="home_position_actuator5" value="182.659103" />
-    <param name="home_position_actuator6" value="302.113647" />
+    <param name="number_of_degree_per_sec" value="30" />
+    <param name="loop_per_command" value="1" />
+    <param name="home_position_actuator1" value="246.00" />
+    <param name="home_position_actuator2" value="203.00" />
+    <param name="home_position_actuator3" value="117.00" />
+    <param name="home_position_actuator4" value="327.00" />
+    <param name="home_position_actuator5" value="188.03" />
+    <param name="home_position_actuator6" value="124.00" />
```

Comment:
If these are to change frequently, they should probably be in their own config file. Otherwise, they will appear in reviews constantly.
Something like what we have for Markhor's flippers comes to mind.

Reply:
I think I prefer having them in the launch file, because I don't think this will change often and we are working on a feature to allow multiple saved positions on the UI. And if we need to have a specific home position for a specific scenario we could create its own launch file for it.